master/internal/rm/actorrm/actor_resource_manager.go

```diff
@@ -212,6 +212,14 @@ func (r *ResourceManager) DeleteJob(
 	return resp, r.Ask(ctx, msg, &resp)
 }
 
+// GetNonDaiJobs returns the details for Non DAI jobs.
+func (r *ResourceManager) GetNonDaiJobs(
+	ctx actor.Messenger,
+	msg sproto.GetNonDaiJobs,
+) (resp []*jobv1.Job, err error) {
+	return resp, r.Ask(ctx, msg, &resp)
```

Comment:
dont forget if you land this and the RMs dont handle `sproto.GetNonDaiJobs` they will implode

Reply:
Do you think I should move this method to Dispatcher RM directly? We can think about moving it up to the actor RM once we have implementations in all types of RMs.

Reply:
You could add an interface like `NonDaiJobsGetter` or something, and where we call it we would to a type assertion to see if the rm satisfies that.. but for now I would just add a default to the actor RM that returns errUnsupported and handle that on calls.